app/models/experiment.js

```diff
@@ -3,14 +3,16 @@ Manage data about one or more documents in the experiments collection
  */
 
 import DS from 'ember-data';
+import JamModel from '../mixins/jam-model';
 
-export default DS.Model.extend({
-  title: DS.attr('string'),
-  description: DS.attr('string'),
-  structure: DS.attr(),  // TODO: Nested document
-  permissions: DS.attr(),
+export default DS.Model.extend(JamModel, {
+    title: DS.attr('string'),
+    description: DS.attr('string'),
+    active: DS.attr('boolean'),
+    structure: DS.attr(),
+    permissions: DS.attr(),
 
-  administrators: DS.hasMany('administrator'),  // TODO: Is Many-to-many correct relationship type?
-  history: DS.hasMany('history'),
-  sessions: DS.hasMany('session'),
+    administrators: DS.hasMany('admin'),
```

Comment:
This relationship would be constructed from the JSONAPI server response, and should represent all admins associated with this particular experiment. (see below: it's not quite implemented due to changing schemas)

For example, if the JSONAPI response explicitly linked to records, it would point towards those links.

Alternately, if the JSONAPI relationships field contained a list of  {type:id} entries, it would ask the adapter to the relevant type for a link to the record ID.

One quirk of our schema is that we store the reference IDs in the `attributes` section, because Jam doesn't inherently construct relationships. Our serializer therefore has some magic to build relationship entries from a list of `relationAttrs` defined on the serializer. 
https://github.com/abought/exp-addons/blob/6c071fd0cee9b8657c6780f3f6c6599a2bee7fee/exp-models/addon/mixins/jam-serializer.js#L41-L41

This PR is a bit confusing because it shows changes that didn't survive the split into the `addons` repo. Actually, I don't think this relationship in this form would do anything- how we handle relations between records has been a bit in flux.

https://github.com/abought/exp-addons/blob/6c071fd0cee9b8657c6780f3f6c6599a2bee7fee/exp-models/addon/serializers/experiment.js
(would need to specify `relationAttrs: ['administrators']` or somesuch.